WS17_18/01_introduction/your assignment/Yatodoa/ViewController.swift

```diff
@@ -0,0 +1,132 @@
+//
+//  ViewController.swift
+//  Yatodoa
+//
+//  Created by Armin Galinowski on 24.10.17.
+//  Copyright © 2017 Armin Galinowski. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+    func getTimestamp() -> String{
+        let now = Date()
+        
+        let uhrzeit = DateFormatter()
+        uhrzeit.timeZone = TimeZone.current
+        uhrzeit.dateFormat = "HH:mm"
+        
+        let tag = DateFormatter()
+        tag.timeZone = TimeZone.current
+        tag.dateFormat="dd.MM.yyyy"
+        
+        
+        let datum = tag.string(from: now)
+        let uhr = uhrzeit.string(from: now)
+        
+        return "erledigt am: " + datum + " um " + uhr + " Uhr"
+    }
+
+    
+    
+    
+    var todos: [String]=[]{
+        didSet{
+            print(todos)
+            
+            if(todos.count==0){
```

Comment:
Bei If-Abfragen ab 3 Fällen würde ich immer Pattern Matchen. Pattern Matching in Swift ist nicht super mächtig, aber ganz OK:

```
switch todos.count {
case 0: "Keine Aufgaben erledigt"
case 1: "1 Aufgabe erledigt"
_: "\(todos.count) Aufgaben erledigt"
}
```

Dadurch ist die Intention schöner zu lesen und es fällt einem einfacher das Ganze zu erweitern